jyulb/internal/common/tests/test_proxy_lattice.py

```diff
@@ -0,0 +1,133 @@
+"""Testing module for jyu-lb proxy-lattice data class."""
+import unittest
+import numpy as np
+import numpy.testing as np_test
+from simphony.core.cuba import CUBA
+from simphony.cuds.lattice import LatticeNode
+from simphony.cuds.abstractlattice import ABCLattice
+from simphony.core.data_container import DataContainer
+from jyulb.internal.common.proxy_lattice import ProxyLattice
+from jyulb.internal.common.domain import PyLattice, PyGeometry
+from jyulb.internal.common.domain import PyIsothermalData
+
+
+class ProxyLatticeTestCase(unittest.TestCase):
```

Comment:
I agree is it not optimal but it is not the template that is at fault here since it strictly applied to the CUDS api description. So until the current CUDS api change (changes approved by the ssb) we cannot claim conformance. I would suggest to open a github task issue in this repo (/simphony-jyulb) on using the template and prioritizing the necessary changes with the SSB.